docs/pages/jpscore/jpscore_inifile.md

```diff
@@ -6,267 +6,1219 @@ sidebar: jupedsim_sidebar
-    - `velocity` (default): color is proportional to speed (slow --> red).
-    - `spotlight`
-    - `group`: color by group
-    - `knowledge`
-    - `router`
-    - `final_goal`
-    - `intermediate_goal`
-
-- `file location` defines the location of the trajectories.
-       All paths are relative to the location of the project file.
-
-- `optional_output`: possibility to give additional output. Set any of these values to `TRUE` (not case sensitive) to get the according value. `FALSE`, any other value and ignoring the option will lead to no output.
-        - `speed`: speed of the pedestrian
-        - `velocity`: x,y components of the pedestrians's velocity
-        - `final_goal`: id of the final goal the pedestrian is heading to
-        - `intermediate_goal`: id of the current goal the pedestrian is heading to (usually a door)
-        - `desired_direction`: x,y components of the pedestrians's desired directions
-        - `group`: group of the pedestrian
-        - `router`: router used by the pedestrian (id accoriding to ini-file)
-        - `spotlight`: pedestrian is highlighted.
+  - `velocity` (default): color is proportional to speed (slow --> red).
+  - `spotlight`
+  - `group`: color by group
+  - `router`
+  - `final_goal`
+  - `intermediate_goal`
+- `file location` defines the location of the trajectories. All paths are relative to the location of the project file.
+- `optional_output`: possibility to give additional output. Set any of these values to `TRUE` (not case sensitive) to
+  get the according value. `FALSE`, any other value and ignoring the option will lead to no output.
+  - `speed`: speed of the pedestrian
+  - `velocity`: x,y components of the pedestrians's velocity
+  - `final_goal`: id of the final goal the pedestrian is heading to
+  - `intermediate_goal`: id of the current goal the pedestrian is heading to (usually a door)
+  - `desired_direction`: x,y components of the pedestrians's desired directions - `group`: group of the pedestrian
+  - `router`: router used by the pedestrian (id accoriding to ini-file)
+  - `spotlight`: pedestrian is highlighted.
 
 ```xml
-<trajectories format="plain"  fps="8">
-<file location="bottleneck_traj.txt" />
-    <optional_output speed="FALSE" velocity="TRUE" final_goal="NoOutputWrongValue"  intermediate_goal="false" desired_direction="true" group="TRUE" router="TRUE" spotlight="TRUE"/>
+
+<trajectories fps="8" precision="2" color_mode="velocity">
+    <file location="trajectories.txt"/>
+    <optional_output speed="FALSE" velocity="TRUE" final_goal="NoOutputWrongValue" intermediate_goal="false"
+                     desired_direction="true" group="TRUE" router="TRUE" spotlight="TRUE"/>
 </trajectories>
 ```
 
-## Train constraints
+### Trajectory Output File
+
+The results of jpscore simulation are written to files in TXT format. This format can be used by other JuPedSim modules.
+
+The provided output format is `plain` (a flat format, just numbers).
```

Comment:
there is no other format anymore. This can be removed. Instead you could refer to the file as "Plain Text"

Reply:
Removed it